### DISCUSSION TITLE
Want to raise a new issue name: Random Password generator

### DISCUSSION THREAD

**Discussion by muditchoudhary** [prathimacode-hub/Awesome_Python_Scripts#309] [Q&A]
Hello I'm @MuditCh57. I want to add a new feature or script (Random Password generator) as LGMSOC 21 participant, Can I raise a issue?

**Comment by prathimacode-hub** [comment]
This issue is already taken. You can take up an another issue. Welcome to the repo once again. @MuditCh57

**Reply by muditchoudhary** [reply]
👍
